administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -232,13 +240,20 @@ COM_LOCALISE_TEXT_TRANSLATION_TRANSLATED="Translated"
 COM_LOCALISE_TEXT_TRANSLATION_UNCHANGED="Unchanged"
 COM_LOCALISE_TEXT_TRANSLATION_UNTRANSLATED="Untranslated"
 COM_LOCALISE_TITLE_TRANSLATION="Translation"
-COM_LOCALISE_TOOLTIP_TRANSLATION_INSERT="<strong>Insert</strong><br>Click to insert"
+COM_LOCALISE_TOOLTIP_TRANSLATION_EXTRA_KEYS_IN_TRANSLATION="If you do not want to keep a string in the translation, you should check the checkbox to remove it on 'Save'.<br> Some strings may not be in reference anymore.<br>Some strings may be specific plural forms that you need to keep."
+COM_LOCALISE_TOOLTIP_TRANSLATION_EXTRA_KEYS_IN_TRANSLATION_ICON="Checking the box this string will be removed from your translation."
```

Comment:
suggest
`"By checking the box this string will be removed from your translation."`